### PR TITLE
Update

### PR DESCRIPTION
**Pull request by kimisan**
1. add a new function, call_hashrate_chart() to append D3.js chart
2. modify the 780G --> 780GH/s

### CODE REVIEW

File: d3module/hashrate/index.html [hunk 1]
```diff
@@ -48,18 +47,18 @@
...
-<script src="http://nvd3.org/assets/js/data/stream_layers.js"></script>
-
 <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/async/2.0.1/async.min.js"></script>
 <script src="bundle_hashrate.js"></script>
 
+<script>
+    call_hashrate_chart();
+</script>
+
 </html>
```
Comment (elaineo) [inline d3module/hashrate/index.html]: this file isn't actually being used... I moved it to public/views/stats/index.html 
So all I need to do is call_hashrate_chart() from the StatsController, right?

Comment (kimisan) [inline d3module/hashrate/index.html]: Yes

Comment (elaineo) [inline d3module/hashrate/index.html]: Where is hashrate.js included? Right now call_hashrate_chart() is undefined

**Comment by elaineo** [comment]
oops, ignore my earlier comments, i forgot to move the files to the proper location. sorry!

**Comment by kimisan** [comment]
haha, 
By the way, for the http://etherscan.io/stats/miner?range=7, if the API is ready, please let me know
And If you have some new visualization idea or assignment , please also let me know (`・ω・´)

**Comment by elaineo** [comment]
i'm sure the ppl on reddit have tons of ideas if you ask them ;)

On Sun, Aug 7, 2016 at 10:41 PM, Kimi Sian-yu Chen <notifications@github.com

**Comment by kimisan** [comment]
haha ,Good idea!

**Pull request merged**